addons/web_editor/static/src/components/media_dialog/image_selector.js

```diff
@@ -141,7 +141,13 @@ export class ImageSelector extends FileSelector {
         await fetch(url).then(async result => {
             const blob = await result.blob();
             blob.id = new Date().getTime();
-            blob.name = new URL(url).pathname.split("/").findLast(s => s);
+            let docHref = this.props.media.ownerDocument.defaultView.location.href;
```

Comment:
Thanks for the clear commit message :+1: 
I tried on website and email marketing and it's working good. But when I try on the project app I still have the issue.
- Go to project app
- do /image
- add `website/static/src/img/phone.png` via the add URL

:arrow_right:  A wrong image is added

Reply:
Indeed, thanks for notifying :pray: The problem comes from the fact that there is not always a replaced image so `this.props.media` can be `undefined` leading to an error. The problem has been solved by giving `window.location.href` as a `base` at URL creation. Note that in the rest of the `image_selector.js` document, `window.location.origin` is sometimes given as a `base` at URL creation; this is not a problem as the `url` argument is `attachment.image_src`. However, in our case, the `url` argument could be a relative source that does not begin with a `/` (so `window.location.href` is given as a `base` at URL creation :+1: )